src/rendezvous/MSC3906Rendezvous.ts

```diff
@@ -64,18 +94,23 @@ export class MSC3906Rendezvous {
     private newDeviceId?: string;
     private newDeviceKey?: string;
     private ourIntent: RendezvousIntent = RendezvousIntent.RECIPROCATE_LOGIN_ON_EXISTING_DEVICE;
+    private v1FallbackEnabled: boolean;
```

Comment:
for me, "fallback" suggests that we try v2 and, if it doesn't work, *fall back* to using v1.

As far as I can tell, once `v1FallbackEnabled` is true, we just use the whole of the v1 flow from that point on, rather than trying v2 at all? (In fact, it looks to me as though "v1 fallback" is *always* enabled, in that as soon as we receive a v1-like payload we switch to v1 mode.)

If so, this is maybe not a great name: `useV1Flow` or something might be better.

Reply:
I'm renaming it to `usingV1Flow` with a comment.